src/main/java/com/google/cloud/genomics/utils/Paginator.java

```diff
@@ -319,6 +321,11 @@ private Jobs(Genomics genomics) {
       SearchReadsResponse,
       Read> {
     
+    private static ImmutableMap<String,String> REQUIRED_STRICT_SHARD_FIELDS =
+        ImmutableMap.<String, String>builder()
+        .put("alignment", ".*\\p{Punct}alignment\\p{Punct}.*") 
```

Comment:
Nevermind the whole thread. These will always be wrapped in the repeated response field, i.e. alignments, annotations, variants